### PR TITLE
Add meeting/users_enable_vote_delegations

### PR DESCRIPTION
**Pull request by reiterl**
Add a migration.

### CODE REVIEW

File: openslides_backend/action/actions/meeting/update.py [hunk 1]
```diff
@@ -114,6 +114,7 @@
     "motion_poll_default_backend",
     "users_enable_presence_view",
     "users_enable_vote_weight",
+    "users_enable_vote_delegations",
```
Comment (r-peschke) [inline openslides_backend/action/actions/meeting/update.py:117]: Please add the new field to the wiki on action meeting.update

**Review by r-peschke — CHANGES_REQUESTED** [review]
Please add the new field to the wiki description for meeting.update, see above

Check this setting in the backend: No forwarding allowed in setting and voting, if the flag is set to false. 
Document the impact of the setting in the various actions.
Otherwise it should be discussed/decided, how to handle this setting.

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Why `users_enable_vote_delegations` is true by default? 
Please set it to false (to prevent migration and to use the same default as `users_enable_vote_weight`). Vote delegation is an optional feature which should enabled by admins who need this. All others should use the simple default.

**Comment by reiterl** [comment]
Okay changed the default of users_enable_vote_delegations to false and remove the migration code (and the test.
Update the data,

File: openslides_backend/action/actions/meeting/update.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/update.py:117]: Add the field to the action wiki

**Review by jsangmeister — APPROVED** [review]

**Review by r-peschke — APPROVED** [review]

**Pull request merged**